### ISSUE TITLE
Improve A11y support

### ISSUE THREAD

**Issue by cassiozen** [tannerlinsley/react-ranger#16]
Hi, I want to help with a PR for a11y - here are the things I would like to tackle:

- Change `getHandleProps` to return by default: `role` ('slider'), `aria-valuemin`, `aria-valuemax` and `aria-valuenow`.
- Change `getHandleProps` to **not** set `tabindex` and **not** set `outline` to none. Ideally, update the examples to use `Buttons` for handles (since they already are interactive elements and have a tab stop set by default).
- Add support for keyboard navigation (When a handle is focused, the user should be able to use left/right arrows to change value)

Do you think these changes make sense for the library?

**Comment by tannerlinsley** [comment]
Absolutely! Go for it :)

**Comment by cassiozen** [comment]
PR: #17
